apps/api/src/queries/buildViewModelForResource/viewModels/photograph.view-model.ts

```diff
@@ -26,14 +26,18 @@ export class PhotographViewModel extends BaseViewModel implements IPhotographVie
      * there.
      */
 
-    constructor(photograph: Photograph, allMediaItems: MediaItem[]) {
+    constructor(photograph: Photograph, _allMediaItems: MediaItem[]) {
         super(photograph);
 
-        const { mediaItemId, photographer } = photograph;
+        const { photographer } = photograph;
 
-        const searchResult = allMediaItems.find(({ id }) => id === mediaItemId);
-
-        this.imageUrl = searchResult?.url;
+        /**
+         * TODO we are now event sourcing this view. We should cache the media
```

Comment:
@yaanahuu2 This may be relevant to your branch.

Reply:
👍